app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/SkillListingPresenter.kt

```diff
@@ -17,13 +16,15 @@ import timber.log.Timber
  *
  * Created by chiragw15 on 15/8/17.
  */
-class SkillListingPresenter : ISkillListingPresenter,
-        ISkillListingModel.OnFetchGroupsFinishedListener, ISkillListingModel.OnFetchSkillsFinishedListener {
+class SkillListingPresenter : ISkillListingPresenter, ISkillListingModel.OnFetchGroupsFinishedListener,
```

Comment:
@arundhati24 make a constructor here which accepts an object of the skillActivity , after this make an object of utilmodel here to solve the string issue